src/site/content/en/blog/houdini-how/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Cross-browser paint worklets and Houdini.how'
+subhead: CSS Houdini worklets and resources are just a few clicks away.
+authors:
+  - una
+date: 2020-12-10
+hero: hero.jpg
```

Comment:
Lightly optimize this with Squoosh if you haven't already, plz

Reply:
I did! :) I can run it again if needed